src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -148,9 +154,16 @@ const ChartsHolder = ({ projections, region, chartId }: ChartsHolderProps) => {
           isMobile={isMobile}
           region={region}
         />
-        <MainContentInner>
-          <RegionVaccinationBlock region={region} />
-        </MainContentInner>
+        {!showVaccinationBlock && (
```

Comment:
We seem to be showing _a_ vaccination block regardless of this boolean.  Perhaps rename it and add a comment to make it clearer what this is actually toggling.

Also, could/should we move the `<MainContentInner>` element outside of this conditional since it seems to be used in both cases?